task05/task05/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task05
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            double x1;
+             P(3, 11,out x1);
+            double x2;
+             P(5, 13, out x2);
+            double x3;
+             P(7, 17, out x3);
+            double x = x1 + x2 + x3;
+            Console.WriteLine(x);
+            Console.ReadKey();
+        }
+        static void P(int a,int b, out double t)
+        {
+            t = (1 + Math.Pow(a, 2)) / (1 + Math.Pow(b, 2));
+        }
```

Comment:
Старайтесь избегать параметров с модификатором `out`. Возвращайте вычисленное значение непосредственно.
Параметры метода следует взять типа `double`.
Квадрат числа проще вычислять умножением.
И зачем вводить дополнительные локальные переменные, если их значения вам нужны только один раз?

А вот этой части задания вы не выполнили:

> результат выведите на консоль с точностью до трех знаков после запятой

```suggestion
        static void Main()
        {
            double x = P(3, 11) + P(5, 13) + P(7, 17);                     
            Console.WriteLine(Math.Round(x, 3));
            
            Console.ReadKey();
        }
        
        static double P(double a, double b)
        {
            return (1 + a * a) / (1 + b * b);
        }
```